src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +32,10 @@ public int findMinEvenNumber(List<String> numbers) {
      * Then return the average of all odd numbers or throw NoSuchElementException.
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return IntStream.range(0, numbers.size())
+                .map(index -> (index % 2 != 0) ? numbers.get(index) - 1 : numbers.get(index))
+                .filter(index -> index % 2 == 1)
+                .average().getAsDouble();
```

Comment:
```suggestion
                .filter(index -> index % 2 == 1)
                .average()
                .getAsDouble();
```
in stream convention is to write each operation from a new line